NicolaeChristof/Rise
# Feature new assets

Added a lot of new assets. PLEASE TEST!!!

How to test:
- Use your earballs and make sure all the sounds are correct, I might have missed a couple because there were a lot of sounds to put in so please make sure everything is correct.

- climb to the top and make sure the squirrel doesn't clip into the new tree model (it will at the top but we are working on that)
